book/website/community-handbook/governance/gov-roles.md

```diff
@@ -0,0 +1,239 @@
+-->
+
+## Chair
+
+> This text is adapted from the "What is the role of the Chairperson" documentation in the  [DIY Committee Guide](https://www.diycommitteeguide.org/) by [Volunteer Now](https://www.volunteernow.co.uk/) and is re-used under a Creative Commons Attribution Non-Commercial Share Alike 2.0. `reformat to reference: https://www.diycommitteeguide.org/what-is-the-role-of-the-chairperson/`
+
+The Chair of a working group, delivery group or the Steering Council provides leadership for the team and facilitates the sustained delivery of the purpose for which the group was created.
+The Chair is responsible for ensuring the group's responsibilities are fairly distributed, and ensuring the broader accountability of the assembled group.
+
+### Responsibilities
+
+There are three dimensions of the Chair's responsibilities:
+
+#### 1. Provide leadership
+
+The Chair articulates a vision for the group and support all members, including new contributors, to make progress towards that goal through individual or group activities.
+
+#### 2. Ensure the group activities align with *The Turing Way* code of conduct and corresponding guiding principles
+
+The Chair works closely with the Secretary `format to reference` to make sure the meetings are planned effectively and conducted in alignment with the working group's charter.
+
+#### 3. Contribute to the leadership of *The Turing Way* community
+
+The Chair of a working group or steering group is a member of the constitutional level decision making Steering Committee for *The Turing Way*.
+The Steering Committee has oversight of the long term strategy for the *Turing Way* community.
+The Chair is responsible for sharing learnings and requests from the Working / Delivery Group to the Steering Committee, and for bringing back requests and guidance from the Steering Committee to WG / DG members.
+
+### Sharing responsibilities
+
+It is up to the Working / Delivery Group to determine if they would like to have one Chair or up to three Co-Chairs sharing the responsibilities listed above.
+
+In *The Turing Way* we have found that having co-chairs works very well in distributing the workload.
+For example, it can remove some of the meeting time requirements and provide more flexibility for meetings if members are connecting from multiple time zones.
+
+We also acknowledge that there is additional work in sharing the responsibilities.
+Two co-chairs do more work than one chair because they have to communicate and align with each other!
+
+We encourage candidates to choose the best option for their personal work circumstances, and their preferred collaboration styles.
+
+### Selection and duration in role
+
+It is up to the working group to determine a process for selecting a Chair or Co-Chair.
+
+It is up to the working group to set a specific time limit on how long an individual can hold the role of Chair or Co-Chair.
+
+Please capture these decisions in the working group's charter.
+
+## Secretary
```

Comment:
I know this is an old established role, but I do wonder whether we could consider alternative names? Secretary is heavily gendered (even if we don't want it to be) and I was thinking that something more neutral like "documentarian" (like the Book Dash working group has) captures the essence of the role clearly enough?

Reply:
I've used "Note taker" or "Records keeper" depending on the scope of the position in some of my groups. I've also noticed that this role is flexible and some groups define it differently then others.